src/Servers/HttpSys/test/FunctionalTests/Utilities.cs

```diff
@@ -134,9 +141,8 @@ internal static IServer CreateDynamicHttpServer(string basePath, out string root
                     root = prefix.Scheme + "://" + prefix.Host + ":" + prefix.Port;
                     baseAddress = prefix.ToString();
 
-                    var server = CreatePump();
+                    var server = CreatePump(configureOptions);
```

Comment:
nit: Can you remove `CreatePump()` with no arguments entirely and just use the new one? That way we don't get confused when to call configureOptions.

Reply:
There are 10 other existing tests that use the existing CreatePump(). If I understand what you're asking for, you want me to replace all those use cases to pass around an empty delegate. Seems weird 🤷‍♂

Reply:
Didn't realize it was called from many other places. Keep it then.